pywinauto/controls/common_controls.py

```diff
@@ -68,7 +66,7 @@
 from ..sysinfo import is_x64_Python
 
 if sysinfo.UIA_support:
-    from ..uia_defines import IUIA
+    from pywinauto.windows.uia_defines import IUIA

```

Comment:
Wow! It's interesting how we have UIA related stuff in Win32 `common_controls.py`. It seems static member `controltypes` is not used for now. I suggest to remove it completely from this file.